Doc/library/asyncio-task.rst

```diff
@@ -785,6 +785,10 @@ Waiting Primitives
    .. versionchanged:: 3.11
       Passing coroutine objects to ``wait()`` directly is forbidden.
 
+   .. deprecated-removed:: 3.11 3.14
```

Comment:
We cannot deprecate behaviour after beta freeze. You need a steering council or release manager exception for this unfortunately

Reply:
This is restoring a behaviour that was supposed to be deprecated

Reply:
Unfortunately, I don't follow. Could you give me some context on what happened?

Reply:
@pablogsal from the original issue report:

> this broke distributed and we missed the DeprecationWarning because ensure_future accepts a wider variety of objects than just futures tasks and coroutines
>
> https://github.com/dask/distributed/issues/6785
>
> the removal happened here https://github.com/python/cpython/commit/903f0a02c16240dc769a08c30e8d072a4fb09154